packages/superset-ui-chart/src/models/ChartPlugin.ts

```diff
@@ -37,13 +37,13 @@ export interface ChartPluginConfig {
   loadChart?: PromiseOrValueLoader<Function>;
 }
 
-export default class ChartPlugin extends Plugin {
+export default class ChartPlugin<T extends FormData = FormData> extends Plugin {
```

Comment:
What does the `= FormData` do?

Reply:
defaults to that generic?

Reply:
Yes, default to that generic so all the charts without specific `FormData` can fallback to it.

Reply:
Aha, got it. That makes sense.